scripts/refresh-assets.ts

```diff
@@ -58,6 +61,29 @@ async function copyIcons() {
 		return { fileName: icon, data: svg.data }
 	})
 
+	// Merge svgs into a single file
+
+	const mergedSvgParts = [
+		mergedIconHeader,
+		'<style>',
+		'svg > [id] { display: none; }',
```

Comment:
comment here would be good on what it's trying to achieve

Reply:
added